tests/test_clinvar.py

```diff
@@ -44,9 +44,14 @@ def test_clinvar(self, fn=data_fn, mod=None):
 
         In other words, you really want to run this with local sequences and UTA.
         """
-        fh = LineIterator(fh=gzip.open(fn) if fn.endswith(".gz") else io.open(fn),
-                          skip_comments=True)
-        for rec in csv.DictReader(fh, delimiter=b"\t"):
+        if sys.version_info < (3, ):
+            fh = LineIterator(fh=gzip.open(fn) if fn.endswith(".gz") else io.open(fn),
+                              skip_comments=True)
+        else:
+            fh = LineIterator(fh=(io.TextIOWrapper(gzip.open(fn), encoding='utf-8')
+                                  if fn.endswith(".gz") else io.open(fn)),
+                              skip_comments=True)
```

Comment:
I didn't run the full `Test_Clinvar` tests because I don't have the local setup to run them quickly. I did run them for a while with no failures.